spec/services/work_version_event_description_builder_spec.rb

```diff
@@ -12,6 +12,9 @@
   let(:work) { work_version.work }
   let(:form) { DraftWorkForm.new(work_version:, work:) }
   let(:attached_files) { [] }
+  let(:filename) { 'xml.svg' }
+  let(:blob) { ActiveStorage::Blob.new(filename:) }
+  let(:attached_file) { AttachedFile.new }
 
```

Comment:
some new mocking since we need the `AttachedFile` objects to respond to blob and filename for validation purposes